<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(250, 218, 221)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="3">
     <widget class="QLabel" name="pic_lab">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Resources.qrc">:/Images/picture.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0" colspan="5">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="2" column="0">
        <widget class="QLabel" name="PasswordLabel">
         <property name="styleSheet">
          <string notr="true">font: 63 40pt &quot;Yrsa&quot;;
color: rgb(236, 83, 83);</string>
         </property>
         <property name="text">
          <string>&amp;Password</string>
         </property>
         <property name="buddy">
          <cstring>PasswordLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="LoginLabel">
         <property name="styleSheet">
          <string notr="true">font: 63 40pt &quot;Yrsa&quot;;
color: rgb(236, 83, 83);</string>
         </property>
         <property name="text">
          <string>&amp;Login</string>
         </property>
         <property name="buddy">
          <cstring>LoginLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="PasswordLineEdit">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
font: 63 40pt &quot;Yrsa&quot;;
color: rgb(236, 83, 83);</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="LoginLineEdit">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
font: 63 40pt &quot;Yrsa&quot;;
color: rgb(236, 83, 83);</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="ShowPasswordCheckBox">
         <property name="styleSheet">
          <string notr="true">font: 63 25pt &quot;Yrsa&quot;;
color: rgb(236, 83, 83);</string>
         </property>
         <property name="text">
          <string>&amp;Show password</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 63 70pt &quot;Yrsa&quot;;
color: rgb(236, 83, 83);</string>
      </property>
      <property name="text">
       <string>MySchool</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1" colspan="3">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 63 35pt &quot;Yrsa&quot;;
color: rgb(236, 83, 83);</string>
      </property>
      <property name="text">
       <string>Study with us!</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" rowspan="2" colspan="5">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="OKButton">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
font: 63 40pt &quot;Yrsa&quot;;
color: rgb(236, 83, 83);</string>
        </property>
        <property name="text">
         <string>            &amp;OK         </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ExitButton">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
font: 63 40pt &quot;Yrsa&quot;;
color: rgb(236, 83, 83);</string>
        </property>
        <property name="text">
         <string>&amp;Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="4">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>816</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ExitButton</sender>
   <signal>clicked()</signal>
   <receiver>LoginWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>805</x>
     <y>587</y>
    </hint>
    <hint type="destinationlabel">
     <x>592</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LoginLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>LoginWindow</receiver>
   <slot>OKButtonSetEnabled()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>808</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PasswordLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>LoginWindow</receiver>
   <slot>OKButtonSetEnabled()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>748</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel">
     <x>812</x>
     <y>388</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OKButtonSetEnabled()</slot>
 </slots>
</ui>
